services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/BaseSpecification.groovy

```diff
@@ -83,7 +83,9 @@ class BaseSpecification extends SpringSpecification implements SetupOnce {
                     .collect { sw -> topologyDefinition.getSwitches().find { it.dpId == sw.switchId } }
 
             nonVirtualSwitches.findAll {
-                it.ofVersion != "OF_12" && !floodlight.getMeters(it.dpId).isEmpty()
+                it.ofVersion != "OF_12" && !floodlight.getMeters(it.dpId).findAll {
+                    it.key > 10 // ids that <=10 are reserved for 'default' meters for default rules
```

Comment:
Now we have 15 reserved `id` for `default` rules.

Reply:
fxd